layouts/partials/blocks/newsletter-topper.html

```diff
@@ -0,0 +1,90 @@
+<section class="mb-6 w-full bg-[#2E2E2E] pt-12">
+  {{ $maxWidth := .Param "max-width" }}
+  {{ $breakpoint := cond (eq $maxWidth "prose") "750px" $maxWidth }}
+  <style>
+    @media (max-width: {{ $breakpoint }}) {
+      .hide-at-max-width {
+        display: none;
+      }
+      .stack-at-max-width {
+        flex-direction: column;
+      }
+      .stack-child-full {
+        width: 100%;
+        padding-right: 0;
+      }
+      .min-h-push {
+        padding-bottom: 4rem;
+      }
+    }
+  </style>
+
+  <div
+    class="{{ if eq $maxWidth "prose" }}
```

Comment:
Since you already have a preamble above defining `$breakpoint`, I would define `$maxWidthClass` above and just do `class="{{ $maxWidthClass }} mx-auto px-4 pb-6"` here.